irida_import/irida_import.py

```diff
@@ -561,39 +578,54 @@ def _add_file(self, added_to_galaxy=None, sample_folder_path=None,sample_folder_
         :return: dataset object or the id of an existing dataset
         """
         galaxy_sample_file_name = sample_folder_path + '/' + sample_file.name
-        if os.path.isfile(sample_file.path):
-            if sample_file.library_dataset_id == None:
-                #grab dataset_id if it does exist, if not will be given False
-                dataset_id = self.existing_file(sample_file.path,galaxy_sample_file_name)
-
-                if dataset_id:
-                    # Return dataset id of existing file
-                    added_to_galaxy = [{'id': dataset_id}]
-                    self.print_logged(time.strftime("[%D %H:%M:%S]:") +
-                                      ' Skipped file with Galaxy path: ' +
-                                      galaxy_sample_file_name)
-                    sample_file.verified = True
-                    self.skipped_files_log.append(
-                        {'galaxy_name': galaxy_sample_file_name})
-                else:
-                    self.logger.debug(
-                        "  Sample file does not exist so uploading/linking it")
-                    added = self.link(
-                        sample_file, sample_folder_id)
-                    if(added):
-                        added_to_galaxy = added
-                        self.print_logged(time.strftime("[%D %H:%M:%S]:") +
-                                          ' Imported file with Galaxy path: ' +
-                                          galaxy_sample_file_name)
-                        self.uploaded_files_log.append(
-                            {'galaxy_name': galaxy_sample_file_name})
-
-                sample_file.library_dataset_id = added_to_galaxy[0]['id']
-
-        else:
-            error = ("File not found:\n Galaxy path:{0}\nLocal path:{1}"
-                     ).format(galaxy_sample_file_name, sample_file.path)
-            raise ValueError(error)
+        file_exists_locally = os.path.isfile(sample_file.path)
+
+        if sample_file.library_dataset_id == None:
+
+            #grab dataset_id if it does exist, if not will be given False
+            dataset_id = self.existing_file(sample_file.path,galaxy_sample_file_name,sample_file.file_size)
```

Comment:
This line would not pass pep8, you need spaces after the comma for arguments. Also stating which argument is what, and each on a new line would be good for readability since there are >2 arguments.
```
self.existing_file(
    sample_file_path=sample_file.path,
    galaxy_name=galaxy_sample_file_name,
    size=sample_file.file_size
)
```

Reply:
Updated in 97c187bc8ab0716b21f78c299f47c20ad4f1ae00